.github/actions/setup-thapi/action.yml

```diff
@@ -0,0 +1,73 @@
+name: 'Setup Thapi Environment'
+runs:
+  using: "composite"
+  steps:
+    - uses: actions/checkout@v4
```

Comment:
Why checking out the repo here? I don't think you use it, and need to recheck different versions later in some actions.

Reply:
Good point!